blog/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    password = models.CharField(max_length=128)
```

Comment:
Redefining the 'password' field in the User model is unnecessary because it is already defined in the AbstractUser model. This can lead to unexpected behavior or security issues. Consider removing this line.